WeakAurasOptions/CommonOptions.lua

```diff
@@ -1113,7 +1113,15 @@ local function PositionOptions(id, data, _, hideWidthHeight, disableSelfPoint, g
         return data.anchorFrameParent or data.anchorFrameParent == nil;
       end,
       hidden = function()
-        return not IsGroupByFrame() and (data.anchorFrameType == "SCREEN" or data.anchorFrameType == "UIPARENT" or data.anchorFrameType == "MOUSE" or IsParentDynamicGroup());
+        return
+          not IsGroupByFrame() and
+          (
+            data.anchorFrameType == "SCREEN" or
```

Comment:
this attribute is getting a bit silly. perhaps we can simplify it? Also, *is* it correct to exclude action button anchors from this option in the first place?